unittests/test_fields.py

```diff
@@ -26,7 +26,7 @@ class FieldTester:
 
 def test_copy_on_write_field():
     class FieldTester:
-        cow = fields.CopyOnWriteField('cow')
+        cow = fields.CopyOnWriteField()
```

Comment:
You can also remove this field completely. It's not used anywhere.

Reply:
Done